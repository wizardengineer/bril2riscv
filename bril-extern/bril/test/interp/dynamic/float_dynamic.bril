@main() {
i: int = const 4;
a: any = const 4;
f: float = const 4;
print i a f; # `4 4 4.00000000000000000`
}