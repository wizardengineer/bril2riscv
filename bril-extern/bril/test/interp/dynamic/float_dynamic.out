4 4 4.00000000000000000
